.github/workflows/container-native.yml

```
@@ -24,6 +24,25 @@ jobs:
           sudo apt install -y crun/testing podman/testing skopeo/testing
           # Something is confused in latest GHA here
           sudo rm /var/lib/containers -rf
+      - name: Download override packages and create yum repo
+        run: |
+          podman run --rm -v $PWD:/run/src -w /run/src quay.io/fedora/fedora:42 sh -c '
```

Let's just use the unversioned moving tag here so we never have to update it. 


```suggestion
          podman run --rm -v $PWD:/run/src -w /run/src quay.io/fedora/fedora:latest sh -c '
```